packages/@aws-cdk/aws-events/test/schedule.test.ts

```diff
@@ -81,4 +69,40 @@ describe('schedule', () => {
       events.Schedule.rate(Duration.seconds(Lazy.number({ produce: () => 5 })));
     }).toThrow(/Allowed units for scheduling/);
   });
-});
+
+  test('rate cannot be a fractional amount of minutes (defined with seconds)', () => {
```

Comment:
Minor nit: since these tests are pretty similar, I suggest you wrap them all in a “describe” block — we do this elsewhere in our tests so you can search it up for an example. Would be good to add in all tests that have to do with rate.